azurerm/internal/services/apimanagement/api_management_resource.go

```diff
@@ -1178,7 +1252,25 @@ func expandApiManagementCustomProperties(d *schema.ResourceData, skuIsConsumptio
 		frontendProtocolSsl3 = v["enable_frontend_ssl30"].(bool)
 		frontendProtocolTls10 = v["enable_frontend_tls10"].(bool)
 		frontendProtocolTls11 = v["enable_frontend_tls11"].(bool)
-		tripleDesCiphers = v["enable_triple_des_ciphers"].(bool)
+
+		// TODO: Remove and simplify after deprecation
+		if v, ok := d.GetOk("security.enable_triple_des_ciphers"); ok {
+			tripleDesCiphers = v.(bool)
+		}
+		if v, ok := d.GetOk("security.triple_des_ciphers_enabled"); ok {
+			tripleDesCiphers = v.(bool)
+		}
```

Comment:
These should be retrieved from the map `v` (if retrieved from the state, they'd need to be `security.0.attr_name`):
```suggestion
		if v, exists := v["enable_triple_des_ciphers"]; exists {
			tripleDesCiphers = v.(bool)
		}
		if v, exists := v["triple_des_ciphers_enabled"]; exists {
			tripleDesCiphers = v.(bool)
		}
```